assets/app.scss

```diff
@@ -0,0 +1,108 @@
+@import '../node_modules/wikimedia-ui-base/wikimedia-ui-base.css';
+@import '../vendor/wikimedia/toolforge-bundle/Resources/assets/wikimedia-base.css';
+
+html,
+body {
+    height: 100%;
+}
+
+body {
+    background-color: var( --wmui-color-base80 );
+    font-size: 0.9rem;
+}
+
+header,
+footer {
+    max-width: var(--width-breakpoint-tablet);
+    margin: 3rem auto;
+}
+
+header {
+    h1 {
+        font-size: 2.25rem;
+        font-weight: 600;
+        letter-spacing: 1px;
+    }
+    nav.user {
+        float: right;
+        text-align: right;
+    }
+}
+
+footer {
+    .toolforge {
+        float: left;
+    }
+    .toolforge-logo {
+        -webkit-filter: grayscale(100%);
+        filter: grayscale(100%);
+    }
+    .toolforge-logo:hover {
+        -webkit-filter: none;
+        filter: none;
+    }
+    .info {
+        text-align: right;
+    }
+}
+
+main,
+aside.tutorial {
+    margin: 1.5rem auto;
+    max-width: var( --width-breakpoint-desktop-wide );
+    padding: 1rem;
+}
+
+main {
+    background-color: var( --background-color-base );
+}
+
+aside.tutorial {
+    border-radius: 2px;
+    border: 1px solid var(--wmui-color-base50);
```

Comment:
`--border-base`

Reply:
By 'done' I of course meant 'not done'... :)

Done *now* though.